src/manifolds/ShapeSpace.jl

```diff
@@ -0,0 +1,241 @@
+
+@doc raw"""
+    KendallsPreShapeSpace{n,k} <: AbstractSphere{ℝ}
+
+Kendall's pre-shape space of ``k`` landmarks in $ℝ^n$ represented by n×k matrices.
+
+# Constructor 
+
+    KendallsPreShapeSpace(n::Int, k::Int)
+"""
+struct KendallsPreShapeSpace{n,k} <: AbstractSphere{ℝ} end
```

Comment:
What about its total space?

Reply:
Should we implement `total_space` if we don't implement it fully as a quotient manifold? Or should we also have `get_orbit_action`? Notice this is for pre-shape space, not shape space.

Reply:
Yes, I meant exactly for `Pre` here. It is a quotient space (though of point could in R^n). For completeness we could maybe add both these functions? I do not see a use directly now, but maybe someone has a use for these (and for considering the PreShape space as a quotient)?

Reply:
This would be quite some work so maybe let's leave that for a future PR?

Reply:
Ok :)